crates/dreammaker/src/lexer.rs

```diff
@@ -71,9 +71,12 @@ table! {
     "!",   Not;
     "!=",  NotEq;
     "\"",  DoubleQuote;
+    "\"\"",  EmptyQuote;
```

Comment:
Now that this has its own token, the parser no longer recognizes it.

Reply:
Oh, I see. I was unsure how the token system actually worked. 
I'll see about using Token::string instead, cause that's bad you're right